### PR TITLE
FIX: Change the link and format of cheatsheet to keep consistency.

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
- [x] Add periods
- [x] Fix links
- [x] Fix typo
- [x] Update the readme

### CODE REVIEW

**Comment by RobPasMue** [comment]
Let me know when I can review @Revathyvenugopal162

**Comment by Revathyvenugopal162** [comment]
Sure @RobPasMue , I see the section containing cheatsheet development using docker, li would like to add add that section link to readme.

**Comment by Revathyvenugopal162** [comment]
@RobPasMue Please review when possible.

File: cheat_sheets/pyprimemesh_cheat_sheet/pyprimemesh_cheat_sheet.tex [hunk 1]
```diff
@@ -114,15 +114,15 @@ \section{\includegraphics[height=\fontcharht\font`\S]{slash.png} Generate Wrappe
 \end{lstlisting}
 
 \section{\includegraphics[height=\fontcharht\font`\S]{slash.png} Generate Surface Mesh}
-Generate Surface Mesh based on min, max size  
+Generate Surface Mesh based on min, max size .
```
Comment (RobPasMue) [inline cheat_sheets/pyprimemesh_cheat_sheet/pyprimemesh_cheat_sheet.tex:117]:
```suggestion
Generate Surface Mesh based on min, max size. 
```

File: cheat_sheets/pyprimemesh_cheat_sheet/pyprimemesh_cheat_sheet.tex [hunk 2]
```diff
@@ -12,7 +12,7 @@
 \raggedright
 \footnotesize
 
-PyPrimeMesh Cheat Sheet
+% PyPrimeMesh Cheat Sheet
 % ----------------------------------------
```
Comment (RobPasMue) [inline cheat_sheets/pyprimemesh_cheat_sheet/pyprimemesh_cheat_sheet.tex:16]: Are you sure about this?

**Review by RobPasMue — APPROVED** [review]
Minor comments. Otherwise LGTM

File: cheat_sheets/pyprimemesh_cheat_sheet/pyprimemesh_cheat_sheet.tex (hunk 2 above)
Comment (Revathyvenugopal162) [reply cheat_sheets/pyprimemesh_cheat_sheet/pyprimemesh_cheat_sheet.tex:16]: Yes, it is rendering on the top left, if you see here , https://[cheatsheets.docs.pyansys.com/pyprimemesh_cheat_sheet.pdf](https://cheatsheets.docs.pyansys.com/pyprimemesh_cheat_sheet.pdf)

File: README.rst [hunk 3]
```diff
@@ -90,3 +97,10 @@ In ``index.rst`` in ``doc/source``, add a new grid item card for your cheatsheet
                :link: https://cheatsheets.docs.pyansys.com/<your_tex_file_name>.pdf
 
 
+Cheat sheets
+~~~~~~~~~~~~~
+
+In the `Documentation <https://cheatsheets.docs.pyansys.com/>`_ page, you can find a comprehensive collection of cheat sheets specifically
```
Comment (Revathyvenugopal162) [inline README.rst:103]:
```suggestion
In the `Documentation <https://cheatsheets.docs.pyansys.com/>`_ page, you can find a comprehensive collection of cheatsheets specifically 
```

File: README.rst [hunk 4]
```diff
@@ -90,3 +97,10 @@ In ``index.rst`` in ``doc/source``, add a new grid item card for your cheatsheet
                :link: https://cheatsheets.docs.pyansys.com/<your_tex_file_name>.pdf
 
 
+Cheat sheets
+~~~~~~~~~~~~~
+
+In the `Documentation <https://cheatsheets.docs.pyansys.com/>`_ page, you can find a comprehensive collection of cheat sheets specifically 
+designed for various PyAnsys products. These cheat sheets are carefully crafted to provide concise and practical information, serving as
```
Comment (Revathyvenugopal162) [inline README.rst:104]:
```suggestion
designed for various PyAnsys products. These cheatsheets are carefully crafted to provide concise and practical information, serving as 
```

File: README.rst [hunk 5]
```diff
@@ -90,3 +97,10 @@ In ``index.rst`` in ``doc/source``, add a new grid item card for your cheatsheet
                :link: https://cheatsheets.docs.pyansys.com/<your_tex_file_name>.pdf
 
 
+Cheat sheets
+~~~~~~~~~~~~~
+
+In the `Documentation <https://cheatsheets.docs.pyansys.com/>`_ page, you can find a comprehensive collection of cheat sheets specifically 
+designed for various PyAnsys products. These cheat sheets are carefully crafted to provide concise and practical information, serving as 
+valuable references for your development or learning journey with PyAnsys. Feel free to post issues and other questions 
+at `PyAnsys cheat sheet issues <https://github.com/ansys/pyansys-cheat-sheet/issues>`_. This is the best place to post questions and code.
```
Comment (Revathyvenugopal162) [inline README.rst:106]:
```suggestion
at `PyAnsys cheatsheet issues <https://github.com/ansys/pyansys-cheat-sheet/issues>`_. This is the best place to post questions and code.
```

File: README.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (Revathyvenugopal162) [reply README.rst]: Done.

**Comment by Revathyvenugopal162** [comment]
Added the comments from review and merging.

**Pull request merged**